lib/suma/member/dashboard.rb

```diff
@@ -14,6 +14,7 @@ def cash_balance
   end
 
   def program_enrollments
-    return @program_enrollments ||= @member.combined_program_enrollments_dataset.active(as_of: @at).all
+    enrollments = @member.combined_program_enrollments_dataset.active(as_of: @at)
```

Comment:
This is confusing to do the work to build the dataset and then ignore it.

```rb
return @program_enrollments if @program_enrollments
@program_enrollments = @member.combined_program_enrollments_dataset.active(as_of: @at).all.sort_by { |pe| pe.program.ordinal }
return @program_enrollments
```

Or just

```rb
return @program_enrollments ||= @member.combined_program_enrollments_dataset.active(as_of: @at).
  all.
  sort_by { |pe| pe.program.ordinal }
```